libs/sdk-ui-ext/src/internal/components/configurationControls/OrientationDropdownControl.tsx

```diff
@@ -18,9 +18,53 @@ export interface IOrientationDropdownControl {
     pushData: (data: IPushData) => void;
 }
 
+export function getAxisesByChartOrientation(properties: IVisualizationProperties) {
```

Comment:
I think `getAxesByChartOrientation` name would fit here more, please rename as the plural form of axis is axes. Also rename on other places in this pull-request.

Reply:
I have updated this. Thanks!

Reply:
@vannguyengd please update also on other places, for example: https://github.com/gooddata/gooddata-ui-sdk/pull/3886/files#diff-0c5f14c6dbd8a379e1fb8626d20b476997f0f2968285c457517da5de74425f30R29

Reply:
Updated! Thank you!